vividus-plugin-web-app-playwright/src/main/java/org/vividus/ui/web/playwright/steps/ElementSteps.java

```diff
@@ -327,4 +343,63 @@ private void saveAttributeValueOfElement(Locator element, String attributeName,
                         () -> softAssert.recordFailedAssertion(
                                 String.format("The '%s' attribute does not exist", attributeName)));
     }
+
+    /**
+     * Checks that the context <b>element</b> has an expected <b>CSS properties</b>
+     * <p>The expected CSS parameters to be defined in the ExamplesTable:</p>
+     * <ul>
+     * <li><b>cssName</b> - the name of the CSS property</li>
+     * <li><b>comparisonRule</b> - String comparison rule: "is equal to", "contains", "does not contain",
+     * "matches".</li>
+     * <li><b>expectedValue</b> - expected CSS property value</li>
+     * </ul>
+     * <p>Usage example:</p>
+     * <code>
+     * <br>Then context element has CSS properties:
+     * <br>|cssName |comparisonRule |expectedValue |
+     * <br>|border  |contains       |solid         |
+     * </code>
+     *
+     * @param parameters The parameters of the expected CSS properties to set as ExamplesTable
+     */
+    @Then("context element has CSS properties:$parameters")
+    public void doesElementHasCssProperties(ExamplesTable parameters)
+    {
+        String getAllCssScript =
+                org.vividus.util.ResourceUtils.loadResource("org/vividus/ui/web/get-element-computed-css-func.js");
+        String script = "([el]) => {" + getAllCssScript + "return getComputedStyleAsMap(el)}";
+        Locator element = uiContext.getCurrentContexOrPageRoot();
+        Map<String, String> elementCss = javascriptActions.executeScript(script, element.elementHandle());
+
+        List<CssValidationResult> cssResults = validateElementCss(parameters, elementCss);
+        attachmentPublisher.publishAttachment("templates/css_validation_result.ftl",
+                Map.of("cssResults", cssResults), "Css validation results");
+    }
+
+    private List<CssValidationResult> validateElementCss(ExamplesTable parameters, Map<String, String> elementCss)
+    {
+        List<Parameters> rowsAsParameters = parameters.getRowsAsParameters();
+        List<CssValidationResult> cssResults = new ArrayList<>();
+        rowsAsParameters.forEach(params ->
+        {
+            Map<String, String> values = params.values();
+            String cssName = values.get("cssName");
+            String expectedValue = values.get("expectedValue");
+            StringComparisonRule comparisonRule = params.valueAs("comparisonRule", StringComparisonRule.class);
+
+            String actualCssValue = getCssValue(elementCss, cssName);
+            boolean passed = softAssert.assertThat(String.format(ELEMENT_CSS_CONTAINING_VALUE, cssName, expectedValue),
```

Comment:
ELEMENT_CSS_CONTAINING_VALUE, its not only containing as you use comparison rule

Reply:
ToDo

Reply:
fixed